src/System.Management.Automation/engine/remoting/common/WireDataFormat/EncodeAndDecode.cs

```diff
@@ -2401,7 +2387,7 @@ internal static RemoteSessionCapability GetSessionCapability(object data)
                 //
                 // So just return the CurrentTimeZone.
 
-#if !CORECLR // TimeZone Not In CoreCLR
+#if !CORECLR // TODO: TimeZone deprecated, need to change to TimeZoneInfo.  Affects serialization, remoting, and backwards compatibility.
```

Comment:
According to documentation here: https://docs.microsoft.com/en-us/dotnet/api/system.timezone?view=netcore-2.0
TimeZone class can still be used to get CurrentTimeZone. A comment about deprecation might not be accurate.

Reply:
At compilation time, dotnet reported it as deprecated

Reply:
Confirm.

Reply:
@SteveL-MSFT 
What do you mean when you say the WSManConfigProvider doesn't work?  Do you have repro steps and errors?

Reply:
@PaulHigin using `sta`, you would get this error:

```none
dir : COM object that has been separated from its underlying RCW cannot be used.
At line:1 char:1
+ dir
+ ~~~
    + CategoryInfo          : NotSpecified: (:) [Get-ChildItem], InvalidComObjectException
    + FullyQualifiedErrorId : System.Runtime.InteropServices.InvalidComObjectException,Microsoft.PowerShell.Commands.GetChildItemCommand
```